src/lib/infra/nginxConfig.ts

```diff
@@ -26,10 +27,69 @@ server {
     proxy_send_timeout ${websocketTimeout};
   }
 
+  error_page 502 /502.html;
+  location /502.html {
+      root ${REMOTE_NGINX_502_PATH};
+  }
+
   location / {
     proxy_pass http://${API_SERVER_HOST_PROD};
   }
+
 }
 
   `.trim();
 };
+
+// Outputs an simple html file for nginx to server during Felt downtime
+export const to502File = (PUBLIC_ADMIN_EMAIL_ADDRESS: string): string => {
+	return `
+  <html>
+    <head>
+    </head>
+    <body style="display: flex; flex-direction: column; justify-content: center;  align-items: center;">
+      <h1>Felt is temporarily down</h1>
+      <h2>If this page persists, please contact ${PUBLIC_ADMIN_EMAIL_ADDRESS}</h2>
+      <div style="white-space: pre-wrap; font-family: monospace;">
+${toASCIIFeltLogo()}
+        </div>
+      </body>
+    </html>
+  `.trim();
+};
+
+//TODO would this be fun to have in felt-util?
+export const toASCIIFeltLogo = (): string => {
```

Comment:
```suggestion
export const toAsciiFeltLogo = (): string => {
```

the convention for acronyms we use is to treat them as words, so `httpThing` and `thingHttp` instead of `hTTPThing` and `thingHTTP`.